src/components/cylc/gscan/GScan.vue

```diff
@@ -133,6 +133,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
                       <workflow-icon
                         :status="scope.node.node.status"
                         :statusMsg="scope.node.node.statusMsg"
+                        :info="scope.node.node"
```

Comment:
Guessing there's a reason we can't add the directive on the `workflow-icon` component directly?

Reply:
Not sure, will check if that's possible.

Reply:
Might be the tooltip, we should remove that now, when you open the menu does it show the workflow status at the top of the menu?

Reply:
It could be, but the menu doesn't include the workflow status.

Reply:
Have a look at https://github.com/cylc/cylc-ui/pull/928/commits/d2dfbd52eb5f8d665defe0a7cc7b2d7f388a10e2 -